scripts/check-idempotency-status

```diff
@@ -7,11 +7,12 @@ import json
 # amount of non-idempotent tests doen't increase
 WHITELIST = {
     'ldap':                 2,
-    'client-base':          3,
+    'client-base':          4,
     'client-orcid-idp':     4,
     'client-google-idp':    2,
     'client-microsoft-idp': 2,
     'client-2fa':           2,
+    'oidc-op':              1,
```

Comment:
noooooo! :( 

What exactly is wrong here?  Which task failed the idempotency checks?  (and where is the interaction with client-base?  it should never be touched.right?)